code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/AEPMessage.java

```diff
@@ -313,10 +317,6 @@ MessageSettings getSettings() {
     /** Invoked after the message is successfully shown. */
     void viewed() {
         isVisible = true;
```

Comment:
listener.onShow(this);
MessagingDelegate delegate = ServiceProvider.getInstance().getMessageDelegate();
if (delegate != null) {
	delegate.onShow(this);
}